docs/Tutorials/Pi3_Cross_Compilation.md

```diff
@@ -160,14 +161,14 @@ python hello_matmul_pi3_generator.py
 python3 hello_matmul_pi3_generator.py
 ```
 
-After this runs, you should see a header file `hello_matmul_pi3.HAT` and an object file `hello_matmul_pi3.o` in the ELF format. The `.HAT` file format is described [here](https://github.com/microsoft/HAT). Together with the `obj`, this is part of a HAT package.
+As we run the script, there should be a header file `hello_matmul_pi3.HAT` and an object file `hello_matmul_pi3.o` in the ELF format. The `.HAT` file format is described [here](https://github.com/microsoft/HAT). This is part of a HAT package with the `obj`.
```

Comment:
should be "`hello_matmul_pi3.hat`"

Reply:
We generally use lowercase for the filenames, e.g. `hello_matmul_pi3.hat` , `*.hat`, but uppercase for the terms "HAT package", "HAT file'.